### PR TITLE
Update withdrawFeeTokens sig

### PR DESCRIPTION
**Pull request by RayXpub**
### Resolves

The `withrawFeeTokens` function does not currently take in any parameters and loops over all the configured fee tokens. If we ever reach a certain number of listed fee tokens, there could be an edge case where we reach gas sizes that are near / exceed the block gas limit on certain chains.

### Solution

Adds a list of `feeTokens` to transfer in the `withdrawFeeTokens` function.

### Note

There is no need to check against the existing list of configured fee tokens as this can be offloaded to the fee aggregator.

### CODE REVIEW

**Review by RensR — DISMISSED** [review]

**Review by elatoskinas — DISMISSED** [review]

File: contracts/src/v0.8/ccip/onRamp/OnRamp.sol [hunk 1]
```diff
@@ -498,9 +498,11 @@ contract OnRamp is IEVM2AnyOnRampClient, ITypeAndVersion, OwnerIsCreator {
   }
 
   /// @notice Withdraws the outstanding fee token balances to the fee aggregator.
-  /// @dev This function can be permissionless as it only transfers accepted fee tokens to the fee aggregator which is a trusted address.
-  function withdrawFeeTokens() external {
-    address[] memory feeTokens = IFeeQuoter(s_dynamicConfig.feeQuoter).getFeeTokens();
+  /// @param feeTokens The fee tokens to withdraw.
+  /// @dev This function can be permissionless as it only transfers tokens to the fee aggregator which is a trusted address.
+  function withdrawFeeTokens(
```
Comment (elatoskinas) [inline contracts/src/v0.8/ccip/onRamp/OnRamp.sol:503]: Should we create an interface for this in `shared`, to standardise the pattern for the fee token withdrawal?

Comment (RensR) [reply contracts/src/v0.8/ccip/onRamp/OnRamp.sol:503]: Not sure shared is the correct place unless you want this interface to be implemented by all other products?

Comment (elatoskinas) [reply contracts/src/v0.8/ccip/onRamp/OnRamp.sol:503]: Yeah, we're aiming to standardise this signature across all integrations

**Review by RensR — DISMISSED** [review]

File: contracts/src/v0.8/ccip/onRamp/OnRamp.sol (hunk 1 above)
Comment (RayXpub) [reply contracts/src/v0.8/ccip/onRamp/OnRamp.sol:503]: Punting this to another PR

**Review by RensR — DISMISSED** [review]

**Review by RensR — DISMISSED** [review]

**Review by jhweintraub — DISMISSED** [review]

**Review by RyanRHall — APPROVED** [review]

**Review by jhweintraub — APPROVED** [review]

**Review by RensR — APPROVED** [review]

**Pull request merged**